tests/cluster/etl/test_simpleloader_upsert.py

```diff
@@ -0,0 +1,99 @@
+from typing import List
+
+from spetlrtools.testing import DataframeTestCase
+
+from spetlr import Configurator
+from spetlr.delta import DbHandle, DeltaHandle
+from spetlr.etl.loaders import SimpleLoader
+from spetlr.utils import DataframeCreator
+from tests.cluster.delta import extras
+from tests.cluster.delta.SparkExecutor import SparkSqlExecutor
+
+
+class UpsertLoaderTests(DataframeTestCase):
```

Comment:
Minor detail but name this test class `SimpleLoaderUpsertTests`. As we have a loader called "UpsertLoader" this could confuse as to what this tests.